maintainers/maintainer-list.nix

```diff
@@ -4448,6 +4448,12 @@
     githubId = 71959829;
     name = "Cleeyv";
   };
+  clementpoiret = {
+    email = "poiret.clement@outlook.fr";
+    github = "clementpoiret";
+    githubId = 10899984;
+    name = "Clement POIRET";
```

Comment:
Is the surname supposed to be all capital letters?

Reply:
It's just a convention in my country, not mandatory. I can change it if you prefer

Reply:
Not at all. I just thought you accidentally capitalized the entire word through a keybinding or similar. Happens to me way too often.

Reply:
Yeah I understand ahah, anyway there are no mistakes here 👍